src/pages/Admin/Venue/VenueWizard/VenueWizard.tsx

```diff
@@ -2,22 +2,25 @@ import React, { useReducer } from "react";
 
 import { useVenueId } from "hooks/useVenueId";
 
+import { AdminRestricted } from "components/atoms/AdminRestricted";
+
 import VenueWizardCreate from "./Create";
-// Components
 import VenueWizardEdit from "./Edit";
-// Reducer
 import { initialState, VenueWizardReducer } from "./redux";
 
 const VenueWizard: React.FC = () => {
   const venueId = useVenueId();
   const [state, dispatch] = useReducer(VenueWizardReducer, initialState);
 
-  if (venueId)
-    return (
-      <VenueWizardEdit venueId={venueId} state={state} dispatch={dispatch} />
-    );
-
-  return <VenueWizardCreate state={state} dispatch={dispatch} />;
+  return (
+    <AdminRestricted>
```

Comment:
You should also add this component in admin v3, that's `Admin_v2.tsx` `AdminAdvancedSettings` and `AdminVenueView`

Reply:
added @ 2d25e8993a1a328bdc9768cf97a372e6f1d8b58c